spec/controllers/assignments_controller_spec.rb

```diff
@@ -334,14 +334,16 @@
 
     context 'when the update succeeds' do
       it 'renders a 200' do
-        post :update, params: { course_id: course.id, id: assignment }.merge(update_params), format: :json
+        post :update, params: { course_id: course.id, id: assignment }
+          .merge(update_params), format: :json
```

Comment:
This breaks up one argument (`params: { course_id: course.id, id: assignment }.merge(update_params)`) into two lines. I suggest just moving the final `format` param to the second line.